src/main/java/com/epam/Application.java

```diff
@@ -0,0 +1,12 @@
+package com.epam;
+
+import com.epam.config.AppConfig;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+public class Application {
+    public static void main(String[] args) {
+        ApplicationContext context = new AnnotationConfigApplicationContext(AppConfig.class);
```

Comment:
Why do we need the `context` variable here? If we do not plan to use it further, we could omit the declaration. IntelliJ Idea static analyzer show this as grey :)